searxng/searxng
# Drop microsoft academic engine

## What does this PR do?

This PR drops the microsoft academic engine from SearXNG.

## Why is this change important?

Microsoft academic was discontinued on 2021-12-31.

Source: https://www.microsoft.com/en-us/research/project/academic/articles/microsoft-academic-to-expand-horizons-with-community-driven-approach/

## How to test this PR locally?

- Install SearXNG
- Check if microsoft academic is available.

## Related issues

https://github.com/searxng/searxng/issues/712
